db/db.go

```diff
@@ -1866,23 +1871,16 @@ func updateQueueDeposits() error {
 func GetQueueAheadOfValidator(validatorIndex uint64) (uint64, error) {
```

Comment:
**praise:** This is a good restructuring to not only reduce the load time but also make it much more readable.

Reply:
thanks ;-)